pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -157,6 +159,25 @@ func New(args runtime.Object, handle frameworktypes.Handle) (frameworktypes.Plug
 		})
 	}
 
+	// check pod by namespace label filter
+	if defaultEvictorArgs.NamespaceLabelSelector != nil {
+		indexName := "metadata.namespace"
+		indexer, err := getNamespacesListByLabelSelector(indexName, defaultEvictorArgs.NamespaceLabelSelector, handle)
+		if err != nil {
+			return nil, err
+		}
+		ev.constraints = append(ev.constraints, func(pod *v1.Pod) error {
```

Comment:
You could probably check whether pod's `namespace` matches `defaultEvictorArgs.NamespaceLabelSelector` directly.